src/encoded/types/target.py

```diff
@@ -21,72 +21,4 @@ class Target(Item):
 
     item_type = 'target'
     schema = load_schema('encoded:schemas/target.json')
-    embedded_list = Item.embedded_list + lab_award_attribution_embed_list + ['targeted_genome_regions.*']
-
-    @calculated_property(schema={
-        "title": "Target summary",
-        "description": "Summary of target information, either specific genes or genomic coordinates.",
-        "type": "string",
-    })
-    def target_summary(self, request, targeted_genes=None, targeted_genome_regions=None,
```

Comment:
Did this just go away? (Maybe it moved somewhere else.) Weird that no tests had to be adjusted. _Are there_ any tests?

Reply:
Yes Andy gave me the ok to get rid of this.